src/easylink/steps/splink/splink_evaluating_pairs.py

```diff
@@ -12,10 +12,13 @@
 blocks_dir = Path(os.environ["BLOCKS_DIR_PATH"])
 diagnostics_dir = Path(os.environ["DUMMY_CONTAINER_DIAGNOSTICS_DIRECTORY"])
 output_path = Path(os.environ["DUMMY_CONTAINER_OUTPUT_PATHS"])
+Path(output_path).parent.mkdir(exist_ok=True, parents=True)
 
 all_predictions = []
 
 for block_dir in blocks_dir.iterdir():
+    if "snakemake" in str(block_dir):
```

Comment:
Is this also checking for the timestamp? Let's change this to be `.startswith(".")` as well if so